README.md

```diff
@@ -68,6 +68,58 @@ You can find the latest version of Saleor, storefront and dashboard in their ind
 - https://github.com/mirumee/saleor-dashboard
 - https://github.com/mirumee/saleor-storefront
 
+## How to solve issues with lack of available space or build errors after update
+
+Most of the time both issues can be solved by cleaning up space taken by old containers. After that, we build again whole platform. 
+
+
+1. Make sure docker stack is not running
+```
+$ docker-compose stop
+```
+
+2. Remove existing volumes
+
+**Warning!** Proceeding will remove also your database container! If you need existing data, please remove only services which cause problems! https://docs.docker.com/compose/reference/rm/
+```
+docker-compose rm
+```
+
+4. Build fresh containers 
+```
+docker-compose build
+```
+
+5. Now you can run fresh environment using commands from `How to run it?` section. Done!
+
+### One-line solution
```

Comment:
Should it be a part of this section? For me, it is more like helpful trick for managing docker containers

Reply:
I'm not sure we should be posting this in our README, it's a hack that will lead to someone losing their database that for some reason had production data.